packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3317,6 +3333,102 @@ class RenderSemanticsAnnotations extends RenderProxyBox {
     markNeedsSemanticsUpdate(onlyLocalUpdates: (value != null) == hadValue);
   }
 
+  /// The handler for [SemanticsAction.tap].
```

Comment:
Can we have more details on each of these? What the semantics of the action are, examples of how the user might trigger it, and so on?

This is the main entry-point to this API so you cannot assume that the user will know about [SemanticsAction].

Reply:
Done.